packages/apollo-cli/src/commands/schema/checkQueries.ts

```diff
@@ -0,0 +1,146 @@
+import 'apollo-codegen-core/lib/polyfills';
+import { Command, flags } from "@oclif/command";
+import * as Listr from "listr";
+import * as path from "path";
+
+import { TargetType, default as generate } from '../../generate';
+
+import { buildClientSchema, validate, findDeprecatedUsages, GraphQLError } from "graphql";
+
+import * as globby from "globby";
+import * as fs from 'fs';
+import { promisify } from 'util';
+
+import { toPromise, execute } from "apollo-link";
+
+import { engineLink, getIdFromKey } from "../../engine";
+import { SCHEMA_QUERY } from "../../operations/schema";
+
+import { loadQueryDocuments } from "apollo-codegen-core/lib/loading";
+
+// TODO: name this check and rename check to diff?
+export default class CheckQueries extends Command {
+  static description = "Generate static types for GraphQL queries.";
+
+  static flags = {
+    help: flags.help({
+      char: "h",
+      description: "Show command help",
+    }),
+    queries: flags.string({
```

Comment:
I'm wondering if we can have a base class or something to extend with these flags in a shared place so we don't have to replicate them everywhere